deployment/scripts/run-local-kubernates-e2e.ps1

```
@@ -0,0 +1,282 @@
+            Warn "Failed to remove hosts entries (elevation declined or write failed, exit code $($proc.ExitCode))."
+        } else {
+            OK "Removed hosts entries."
+        }
+    }
+    Remove-WslHostsEntry
+}
+
+function Add-WslHostsEntry {
+    # WSL auto-generates its own /etc/hosts (separate from Windows), and
+    # test-em-all.sh runs inside bash/WSL, so it needs its own entry too.
+    if (bash -lc "grep -q retailstore.local /etc/hosts" 2>$null) {
```

PSScriptAnalyzer (1.25.0)

[warning] Did you mean to use the redirection operator '>'? The comparison operators in PowerShell are '-gt' (greater than) or '-ge' (greater or equal).

(PSPossibleIncorrectUsageOfRedirectionOperator)